src/actions.rs

```diff
@@ -154,95 +154,91 @@ pub fn remove_filtered_policy(
     let field_values = normalize_casbin_rule(field_values, field_index);
 
     let boxed_query = if field_index == 5 {
-        diesel::delete(casbin_rule.filter(ptype.eq(pt).and(eq_empty!(&field_values[0], v5))))
+        diesel::delete(casbin_rule)
+            .filter(ptype.eq(pt).and(v5.is_null().or(v5.eq(&field_values[0]))))
             .into_boxed()
     } else if field_index == 4 {
-        diesel::delete(
-            casbin_rule.filter(
+        diesel::delete(casbin_rule)
+            .filter(
                 ptype
                     .eq(pt)
-                    .and(eq_empty!(&field_values[0], v4))
-                    .and(eq_empty!(&field_values[1], v5)),
-            ),
-        )
-        .into_boxed()
+                    .and(v4.is_null().or(v4.eq(&field_values[0])))
+                    .and(v4.is_null().or(v5.eq(&field_values[1]))),
```

Comment:
I think there's a typo here and it should be `v5.is_null()`.

Reply:
@peschkaj